aspnet/migration/rc1-to-rc2.rst

```diff
@@ -0,0 +1,352 @@
+
+ASP.NET 5 RC1 apps were based on the .NET Execution Environment (DNX) and made use of DNX specific features. ASP.NET Core RC2 is based on .NET Core, so you must first migrate your application to the new .NET Core project model. For information on migrating from DNX to .NET Core CLI see <need link>.
+
+For information regarding a complete list of breaking changes in RC2 for ASP.NET Core, see the `ASP.NET Core Announcements page <https://github.com/aspnet/announcements/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0-rc2>`_
+
+Creating your web application host
+----------------------------------
+
+Since ASP.NET Core apps are just console apps, you must define an entry point for your application in ``Program.Main`` that sets up a web host, then tells it to start listening. Below is an example of the startup code for the built-in `Web Application` template in Visual Studio.
+
+.. code-block:: c#
+
+  public class Program
+  {
+    public static void Main(string[] args)
+    {
+        var host = new WebHostBuilder()
+            .UseKestrel()
+            .UseContentRoot(Directory.GetCurrentDirectory())
+            .UseDefaultHostingConfiguration(args)
+            .UseIISIntegration()
+            .UseStartup<Startup>()
+            .Build();
+
+        host.Run();
+    }
+  }
+
+
+Configure the webroot and port in the ``hosting.json`` file. The web root of your application is now defined when setting up the web host and defaults to ``wwwroot``.
+
+Additionally, you must turn on server garbage collection in ``project.json`` or, ``app.config`` when running ASP.NET projects on the full .NET framework.
+
+.. code-block:: c#
+
+  {
+    "runtimeOptions": {
+      "gcServer": false,
+      "gcConcurrent": true
+    }
+  }  
+
+You can find more information about Garbage Collection configuration at: https://github.com/aspnet/Announcements/issues/175
+
+WebHostBuilder API updates
+--------------------------
+
+All classes prefixed with WebApplication have been renamed to WebHost. This includes:
+
+===========================    =========================
+IWebApplicationBuilder         IWebHostBuilder
+WebApplicationBuilder          WebHostBuilder
+IWebApplication                IWebHost
+WebApplication                 WebHost
+WebApplicationOptions          WebHostOptions
+WebApplicationDefaults         WebHostDefaults
+WebApplicationService          WebHostService
+WebApplicationConfiguration    WebHostConfiguration
+===========================    =========================
+
+The ``commands`` section of ``project.json`` has been removed completely. Use ``run`` or ``dotnet <dllname>`` instead.  
+
+
+IHostingEnvironment changes 
+---------------------------
+
+All environment variables are now prefixed with the ``ASPNETCORE_`` prefix.
+
+======================  =========================    
+Old prefix              New prefix                           
+======================  =========================  
+ASPNET_WEBROOT          ASPNETCORE_WEBROOT
```

Comment:
Where did you get this list? I think you may be missing some keys. Take a look at https://github.com/aspnet/Hosting/blob/4cad0a2f017a1e200d5ddc4895cde9f813578ebd/src/Microsoft.AspNetCore.Hosting.Abstractions/WebHostDefaults.cs